uk.co.compucorp.civicrm.hrleaveandabsences/sql/auto_uninstall.sql

```diff
@@ -12,17 +12,3 @@ DROP TABLE IF EXISTS `civicrm_hrleaveandabsences_work_week`;
 DROP TABLE IF EXISTS `civicrm_hrleaveandabsences_work_pattern`;
 DROP TABLE IF EXISTS `civicrm_hrleaveandabsences_absence_period`;
 DROP TABLE IF EXISTS `civicrm_hrleaveandabsences_public_holiday`;
-
--- Deletes sample data for contracts (for l&a, the drop tables already take care of the job)
-SET foreign_key_checks = 0;
```

Comment:
I think there are some references to old tables in this file that we should remove too. e.g `civicrm_hrleaveandabsences_sickness_request`, `civicrm_hrleaveandabsences_sickness_request`